### PR TITLE
[mis_builder] Added column level CSS report

### PR DESCRIPTION
**Pull request by nmoturi**
This will add a column level CSS support.
unnamed

### CODE REVIEW

File: mis_builder/models/mis_builder.py [hunk 1]
```diff
@@ -41,10 +41,6 @@
 from .aggregate import _sum, _avg, _min, _max
 
 _logger = logging.getLogger(__name__)
-DATE_LENGTH = len(datetime.date.today().strftime(
-    tools.DEFAULT_SERVER_DATE_FORMAT))
-DATETIME_LENGTH = len(datetime.datetime.now().strftime(
-    tools.DEFAULT_SERVER_DATETIME_FORMAT))
```
Comment (lmignon) [inline mis_builder/models/mis_builder.py:47]: You must preserve this code and keep the method '_utc_midnight' unchanged

File: mis_builder/models/mis_builder.py [hunk 2]
```diff
@@ -606,11 +600,11 @@ def _fetch_queries(self, cr, uid, c, context):
                     cr, uid, obj_ids, field_names, context=context)
                 s = AutoStruct(count=len(data))
                 if query.aggregate == 'min':
-                    agg = _min
+                    agg = min
```
Comment (lmignon) [inline mis_builder/models/mis_builder.py]: Plz revert this change

File: mis_builder/models/mis_builder.py [hunk 3]
```diff
@@ -606,11 +600,11 @@ def _fetch_queries(self, cr, uid, c, context):
...
                 elif query.aggregate == 'max':
-                    agg = _max
+                    agg = max
```
Comment (lmignon) [inline mis_builder/models/mis_builder.py]: Same here

File: mis_builder/models/mis_builder.py [hunk 4]
```diff
@@ -606,11 +600,11 @@ def _fetch_queries(self, cr, uid, c, context):
...
                 elif query.aggregate == 'avg':
-                    agg = _avg
```
Comment (lmignon) [inline mis_builder/models/mis_builder.py:613]: Same here

**Comment by nmoturi** [comment]
Thanks Laurent, I will check if we have latest code and push it again.

**Comment by laetitia-gangloff** [comment]
@nmoturi Why did you revert all your commit ?!
Is this PR always on ?

**Comment by sbidoul** [comment]
Note we are planning an evolution of mis_builder to add a proper notion of styles for rows and cells. The goal is to configure styles in a single place, and to allow preserving styles when exporting to Excel. I'll see if it works to add column styles too.

**Comment by sbidoul** [comment]
Preserved as an enhancement suggestion in https://github.com/OCA/mis-builder/issues/14
